generator/src/main/java/com/scottlogic/deg/generator/StandardGenerationEngine.java

```diff
@@ -31,5 +37,10 @@ public void generateDataSet(Profile profile, GenerationConfig config, OutputTarg
             this.dataGenerator.generateData(profile, analysedProfile.getMergedTree(), config);
 
         outputTarget.outputDataset(generatedDataItems, profile.fields);
+
+        if (monitor instanceof VelocityMonitor) {
+            monitor.reportVelocity(0);
```

Comment:
This should be an explicit method e.g. `reportEndOfGeneration()` and the `System.out` call present in this method.
Currently providing `--quiet` will emit this message to the output.